conduct.md

```diff
@@ -52,35 +52,35 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is key to both successful and sustainable code. However it's important that all communication between students and team members follows the three pillars of Respect, Trust, and Humility. In general, students will *respect* their peers, *trust* their contributions and thoughts, and practice *humility* in all aspects of communication, in person, on GitHub, in the Discord, and otherwise.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should communicate in a professional and respectful manner.
+- Students should use words like "Please" and "Thank you".
+- Students will allow other members and teams speak freely.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not communicate in a way that offends or disrespects others.
+- Students should not interrupt others - we all have something to share.
+- Students should not show signs of passive aggressive behavior. Working as a team means keeping emotions out of work.
 
 ### Participation
 
-TODO: Write an introductory paragraph.
+Participation for students will be a multitude of factors. We all have bad days and  we all have topics we know more or less about but how many times a student speaks a day will not solely make up their participation. It is being prepared each day, having done the homework, coming to class with an open mind to learn and making an effort to be apart of either small group or large group conversations that will reflect students participation.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students will respect all peers opinions.
+- Students will show up to every class ready to learn and engage with the class.
+- Students participation will be reflected in participation surveys.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students will not interrupt their peers.
+- Students will not skip reading or assignments in general.
```

Comment:
The "in general" seems to suggest that it's okay the skip reading *some* of the time.
Since reading assignments are most critical to being able to participate effectively in class, could this be reworded to state that students should not miss reading assignments?